andela/ah-kgl-avengers-frontend
# #165020156 Setup Redux

### What does this PR do?
This PR Set Up the Redux to help in state management of our application.
### Description of Task to be completed?
- Installing the Redux and react-redux libraries
- Setup the actions, reducers, and store.
- Create the folder structure for the project
### How should this be manually tested?
 - Clone the Repo and checkout into the `ch-setup-redux-165020156` branch
  - Run `npm install` to install the dependencies.
  - Run `npm start` and check in the browser, You will get a message: `Welcome to Authors Haven` from the API.
### Any background context you want to provide?
   - You have to add the `API`url to your .env file.
### What are the relevant pivotal tracker stories?
The story: [165020156](https://www.pivotaltracker.com/n/projects/2321840)
### Screenshots (if appropriate)

### Questions:
